Makefile

```diff
@@ -223,3 +223,14 @@ devenv-up: $(INSTALL_TARGET) $(EMILY_LAMBDA_BINARY) $(EMILY_CDK_TEMPLATE)
 devenv-down:
 	cd devenv/local/docker-compose && \
 	CONTAINER_HOST=$(_CONTAINER_HOST) sh down.sh
+
+run-signer:
+	docker compose --file docker-compose.signer.yml down;
+	docker compose --file docker-compose.signer.yml up postgres bitcoind --detach;
+	POSTGRES_PORT="0"; \
+	while [ "$$POSTGRES_PORT" -le 0 ]; do \
+		sleep 1; \
+		POSTGRES_PORT=$$(docker port sbtc-postgres 5432 | awk -F: '{print $$2}'); \
```

Comment:
Not sure what I did wrong but the following returned an empty string for me:
```bash
docker port postgres 5432 | awk -F: '{print $$2}'
```
maybe try this?
```bash
docker port postgres 5432 | cut -d ":" -f 2
```

Reply:
Weird, works perfectly for me. If you change it locally does it work for you?

Reply:
Per slack we'll leave it as-is for now. If you end up using this and need to change it then feel free, both work for me.